code/src/components/Header.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import DayJS from 'react-dayjs';
+
+import TaskSummary from './TaskSummary';
+
+const Container = styled.div`
+width: 100%;
+padding: 30px 0;
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+align-items: flex-start;
+background-color: #F4F4F4;
+border-radius: 15px;
+@media (min-width: 768px){
+  width: 680px;
+}
+`;
+const DateContainer = styled.div`
+margin-left: 7px;
+display: flex;
+flex-direction: column;
+font-family: 'Noto Sans KR', sans-serif;
+@media (min-width: 768px){
+  font-size: 22px;
+}
+`;
+const Header1 = styled.h1`
+margin: 0;
+font-family: 'Rammetto One', cursive;
```

Comment:
When loading the app in the browser the h1 is set to cursive for a few seconds then changes. Not sure if this is intentional or if you wanted to stick to one font style only ?

Reply:
I tried replacing the fallback font to another more similar to Rammetto One, but not sure it works...